meta/main.yml

```diff
@@ -0,0 +1,17 @@
+---
+galaxy_info:
+  author: Juwai Limited
+  description:
+  company: Juwai Limited
+  license: MIT / BSD
+  min_ansible_version: 1.9
+  platforms:
+  - name: EL
+    versions:
+    - 6
+  - name: Amazon
```

Comment:
I might need to remove the Amazon platform here. Galaxy didn't recognize it last time I imported a role.